pepper-apis/studybuilder-cli/studies/osteo/patches/germline-consent-addendum-ped-v3.conf

```diff
@@ -0,0 +1,135 @@
+          },
+          "bodyTemplate": {
+            "TemplateType": "HTML",
+            "templateText": """
+                            <p>$study_contact_pediatric</p>
+                            <ul>
+                                 <li>$study_contact_li2_pediatric</li>
+                            </ul>
+                            <p>$paragraph_contact_representative_pediatric</p>
+                            """,
+            "variables": [
+              {
+                "name": "study_contact_pediatric",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": """If you have questions about the study, please contact the research doctor or study staff listed below by emailing  <a href="mailto:info@lmsproject.org" class="Link">info@lmsproject.org</a> or calling <a href="tel:651-403-5556" class="Link">651-403-5556</a>:"""
+                  }
+                ]
+              },
+              {
+                "name": "study_contact_li2_pediatric",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": "Katie Janeway, MD"
+                  }
+                ]
+              },
+              {
+                "name": "paragraph_contact_representative_pediatric",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": "For questions about your rights as a patient, please contact a representative of the Office for Human Research Studies at (617)-632-3029. This can include questions about your participation in the study, concerns about the study, a research related injury, or if you feel/felt under pressure to enroll in this research study or to continue to participate in this research study. Please keep a copy of this document in case you want to read it again."
+                  }
+                ]
+              }
+            ]
+          },
+          "blockType": "CONTENT",
+          "shownExpr": null
+        }
+      "old_template_search_text": "$study_contact_pediatric</p>"
+    }
+
+  ],
+  "block-add-parent":
+    {
+      "titleTemplate": {
+        "templateType": "HTML",
+        "templateText": """<h3 class="activity-question-title">$germline_consent_pediatric_date_title_parent</h3>""",
+        "variables": [
+          {
+            "name": "germline_consent_pediatric_date_title_parent",
+            "translations": [
+              { "language": "en", "text": "Date" }
+            ]
+          }
+        ]
+      }
+      "bodyTemplate": {
+        "templateType": "HTML",
+        "templateText": """<p class="paragraph">$ddp.date("MM / dd / yyyy")</p>""",
+        "variables": []
+      },
+      "blockType": "CONTENT",
+      "shownExpr": """user.studies["cmi-osteo"].forms["GERMLINE_CONSENT_ADDENDUM_PEDIATRIC"].questions["ADDENDUM_CONSENT_BOOL_PEDIATRIC"].answers.hasTrue()"""
+    }
+
+  "block-add-child":
```

Comment:
Date was incorrectly coded as user input where it should be a system date.
corrected in this revision
disabled the input Date question and added as a system date